packages/flutter/lib/src/material/material_theme.dart

```diff
@@ -0,0 +1,131 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
If this PR is just for the scaffold the `MaterialTheme` changes shouldn't be included. Isn't this duplicating #115273?

Reply:
I noted in the description that this depends on https://github.com/flutter/flutter/pull/115273. Does GitHub support diffbase?

Reply:
[not the answer you are looking for] I think in this case, it would be best not to land the MaterialTheme at all, just the ScaffoldTheme

Reply:
SG, landing in a different way, closing this PR